src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -435,23 +435,12 @@ public function testDecodeInvalidXml()
         $this->encoder->decode('<?xml version="1.0"?><invalid><xml>', 'xml');
     }
 
+    /**
+     * @expectedException \Symfony\Component\Serializer\Exception\UnexpectedValueException
+     */
     public function testPreventsComplexExternalEntities()
     {
-        $oldCwd = getcwd();
-        chdir(__DIR__);
```

Comment:
I'm not 100% sure, but I think that it's to test that file inclusion is blocked (the `XmlEncoderTest.php` must be in the current directory for this test to be correct).

As this is related to security, I would keep this line.